### PR TITLE
[jax2tf] Implemented conversion for triangular_solve.

### PR DESCRIPTION
**Pull request by bchetioui**

### CODE REVIEW

File: jax/experimental/jax2tf/jax2tf.py [hunk 1]
```diff
@@ -1718,6 +1718,30 @@ def _eigh(operand: TfVal, lower: bool):
 
 tf_impl[lax_linalg.eigh_p] = _eigh
 
+def _triangular_solve(a: TfVal, b: TfVal, *, left_side: bool, lower: bool,
+                      transpose_a: bool, conjugate_a: bool,
+                      unit_diagonal: bool):
+  if unit_diagonal:
+    a = (a - tf.linalg.diag(tf.linalg.diag_part(a)) +
```
Comment (hawkinsp) [inline jax/experimental/jax2tf/jax2tf.py:1725]: This would not be correct if there are NaNs on the diagonal.

**Review by hawkinsp — COMMENTED** [review]
One small nit.

File: jax/experimental/jax2tf/jax2tf.py (hunk 1 above)
Comment (bchetioui) [reply jax/experimental/jax2tf/jax2tf.py]: Thanks Peter! `d840c34` should fix this!

**Review by hawkinsp — APPROVED** [review]

**Comment by cghawthorne** [comment]
Do you need to remove `lax_linalg.triangular_solve_p` from `tf_not_yet_impl`?

**Comment by gnecula** [comment]
> Do you need to remove `lax_linalg.triangular_solve_p` from `tf_not_yet_impl`?

Good catch. There is a presubmit test that will fail because of this. We just don't run that test in GitHub

**Comment by bchetioui** [comment]
Removed it. Thanks @cghawthorne :)

**Pull request merged**